FittedSheets/SheetContentViewController.swift

```diff
@@ -126,6 +139,45 @@ public class SheetContentViewController: UIViewController {
         self.contentWrapperView.layer.cornerRadius = self.treatPullBarAsClear ? 0 : self.cornerRadius
         self.childContainerView.layer.cornerRadius = self.treatPullBarAsClear ? self.cornerRadius : 0
     }
+
+    private var titleBarViewTopInset: CGFloat {
+        return (self.options.pullBarHeight + self.gripSize.height) / 2 + 2
+    }
+
+    private func setupTitleBarView() {
+        guard self.options.shouldExtendBackground == false else { return }
```

Comment:
The titleBar isn't compatible with the `shouldExtendBackground` option. We don't use this option (and have no plan to). Ideally we'd remove it, but that would mean substantial changes to the library that will prevent pulling in future updates (if that's ever needed)